sql/api/src/main/scala/org/apache/spark/sql/catalyst/util/SparkDateTimeUtils.scala

```diff
@@ -36,12 +37,14 @@ trait SparkDateTimeUtils {
 
   final val TimeZoneUTC = TimeZone.getTimeZone("UTC")
 
+  final val tzRegexShort = Pattern.compile("(\\+|\\-)(\\d):")
+  final val tzRegexLong = Pattern.compile("(\\+|\\-)(\\d\\d):(\\d)$")
```

Comment:
Could you assign more specific names like:
```suggestion
  final val singleHourTz = Pattern.compile("(\\+|\\-)(\\d):")
  final val singleMinuteTz = Pattern.compile("(\\+|\\-)(\\d\\d):(\\d)$")
```

Short and Long term slightly confuses sine they could not reflect actual tz for example:
Long: 08:3
Short: 8:30:00